rosie/rosie/chamber_of_deputies/tests/test_adapter.py

```diff
@@ -82,3 +82,11 @@ def test_update(self, reimbursements, fetch):
             call()()
         ))
         fetch.assert_called_once_with(Adapter.COMPANIES_DATASET, self.temp_path)
+
+    @freeze_time('2010-11-12')
```

Comment:
Do we need to freeze time here? If we're reading from fixtures, _maybe_ we don't…

Reply:
I'll test without the freeze time so. Thanks